docs/self-managed/troubleshooting/platform-deployment.md

```diff
@@ -0,0 +1,17 @@
+---
+id: platform-deployment
+title: "Platform deployment"
+---
+
+## Connection issues between Identity and Keycloak
+
+By default, Keycloak requires all non-local IP ranges to use SSL. It has been noted that in certain
+cloud environments where communication between containers is performed using IPs outside the local range,
+components are not able to function correctly.
+
+To resolve this, we would suggest disabling the requirement for SSL connections by following the steps
+in the [SSL modes](https://www.keycloak.org/docs/16.1/server_admin/#_ssl_modes) section of the Keycloak
+docs and setting the `Require SSL` field value to `none` for both the `master` and `camunda-platform` realms.
```

Comment:
The default solution should be to configure the ip range for containers to be "local", so this section should contain the specific IP ranges. As a fallback (only if configuring local IP ranges is not possible) we can then suggest disabling SSL requirements in keycloak. It has to be noted that this is a two step process:
1. disable SSL on `master` realm, restart Identity
2. disable SSL on Camunda realm, restart Identity